packages/plugin-vertical-embed/web/src/components/styles.scss

```
@@ -0,0 +1,30 @@
+
+.card {    
+    :global {
+        .vertical-embed-card {            
+            border-color: rgba(var(--ricos-text-color-tuple), 0.2);
+            background-color: var(--ricos-background-color);
+        }
+
+        .vertical-embed-content {        
+            .vertical-embed-title {
+                font-family: var(--ricos-custom-h2-font-family);
+                color: var(--ricos-text-color);
+            }
+
+            .vertical-embed-subtitle, .vertical-embed-buttonText {
+                font-family: var(--ricos-custom-p-font-family);
+                color: var(--ricos-text-color);
+            }
+
+            .vertical-embed-buttonText {
+                color: var(--ricos-background-color);
+            }
+
+            .vertical-embed-button {
+                background-color: var(--ricos-action-color);
```

Add this before merge (we now support custom button color - #1808 )
```suggestion
                background-color: var(--ricos-custom-button-color, var(--ricos-action-color));
```

---

🚀